actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Raise an `ArgumentError` when invalid `:on` or `:except` options are passed into `#resource` and `#resources`.
```

Comment:
I think you mean `:only`, not `:on`.

Reply:
Whoops, thank you!

https://github.com/rails/rails/pull/53214